apps/mech/common-util/hooks/useHelpers.ts

```diff
@@ -1,15 +1,14 @@
 import { isNumber } from 'lodash';
 import { useMemo } from 'react';
 import { useSelector } from 'react-redux';
-import { useAccount, useChainId } from 'wagmi';
+import { useAccount } from 'wagmi';
 
+import type { RootState } from 'store/types';
 import { getIsValidChainId } from '../functions';
 
 export const useHelpers = () => {
-  // @ts-ignore TODO: add types
-  const account = useSelector((state) => state?.setup?.account);
-  // @ts-ignore TODO: add types
-  const chainId = useSelector((state) => state?.setup?.chainId);
+  const account = useSelector((state: RootState) => state?.setup?.account);
```

Comment:
we should ideally use useAppSelector approach similar to what we have in gover, launch or contribute, so that we only import useAppSelector and types related to this particular store are already there